draft-ietf-jsonpath-base.md

```diff
@@ -122,17 +116,15 @@ normative:
 --- abstract
 
 JSONPath defines a string syntax for identifying values
-within a JavaScript Object Notation (JSON) document.
-
---- note_Contributing
+within a JavaScript Object Notation (JSON, RFC 8259) value.
 
-This document picks up the popular JSONPath specification dated
-2007-02-21 and provides a normative definition for it.
-In its current state, it is a strawman document showing what needs to
-be covered.
+--- to_be_removed_note_Discussion_Venues
```

Comment:
Not sure what this means. Is the intention to merge this PR or leave it open as a way of capturing the discussion at the IETF 112 meeting?

Reply:
If you want to see the effect of this:

https://ietf-wg-jsonpath.github.io/draft-ietf-jsonpath-base/tim-note-intro/draft-ietf-jsonpath-base.html

These are meant to be merged quickly, so we can have a version submitted on Nov 7 that addresses all of Tim's comments.  If there is anything controversial, we can discuss that at the IETF, but I'm sure we don't want to play the "delay merging until doomsday" game any longer.